backend/src/main/java/io/papermc/hangar/db/dao/v1/VersionsApiDAO.java

```diff
@@ -224,13 +224,13 @@ SELECT date, sum(platform_downloads) vs_totalDownloads, array_agg((ARRAY[subquer
                    project_versions pv,
                    (SELECT generate_series(:fromDate::date, :toDate::date, INTERVAL '1 DAY') AS day) dates
                        LEFT JOIN project_versions_downloads pvd ON dates.day = pvd.day
-              WHERE lower(p.slug) = lower(:slug)
-                AND pv.version_string = :versionString
+              WHERE p.id = :projectId
```

Comment:
can remove projects here

Reply:
Is the `pvd.project_id = p.id` check here even necessary? shouldn't verifying the version already guarantee that the project id is equal?